src/guiFormSpecMenu.cpp

```diff
@@ -1674,8 +1698,16 @@ void GUIFormSpecMenu::parseElement(parserData* data, std::string element)
 	std::string type = trim(parts[0]);
 	std::string description = trim(parts[1]);
 
+	if (type == "container") {
+		parseContainer(data, description);
```

Comment:
`return;`
Otherwise it would throw an error on the bottom of the function: `Unknown DrawSpec: type=..`